### DISCUSSION TITLE
PLZ Update this and add two gyro BMI270 & ICM42688P

### DISCUSSION THREAD

**Discussion by MaxCubeX** [emuflight/EmuFlight#880] [General]
PLZ Update this release and add two gyro BMI270 & ICM42688P

Because when installing the firmware on the Diatone Mamba MK4 F722 APP, the gyroscope does not work

Add support for MAMBA F722_2022A

**Comment by nerdCopter** [comment]
needs testing: https://github.com/emuflight/EmuFlight/issues/923#issuecomment-1709024218
